docs/metrics/lead-time-reviews.md

```diff
@@ -1,48 +1,92 @@
-| Commits  | Number of commits in a pull request, even if the pull request is squashed. |
-| Reviews  | Number of reviews in a pull request.                                       |
-| Comments | Number of comments and review comments in a pull request.                  |
+The following metrics provide insights into the level of engagement and efficiency of your code review process:
+
+-   [Time to first review](#time-to-first-review)
+-   [Time to first approval](#time-to-first-approval)
+-   [Time from first to last approval](#time-from-first-to-last-approval)
+-   [Time from last approval to merge](#time-from-last-approval-to-merge)
+-   [Outlier review details](#outlier-review-details)
+
+!!! note
+    Pulse calculates these metrics based on pull request data and:
+
+    -   Considers that reviews are approvals, change requests, and inline pull request comments, but not pull request conversation comments
+    -   Excludes reviews and approvals that are done after the pull requests are merged.
+
+### Time to first review
+
+How long does it take to have the first review on a pull request:
+
+```text
+first review timestamp - pull request open timestamp
+```
+
+### Time to first approval
+
+How long does it take to have the first approval on a pull request:
+
+```text
+first approval timestamp - pull request open timestamp
+```
+
+### Time from first to last approval
+
+How long does it take between the first and last approvals on a pull request:
 
-### Time to first/last approve
+```text
+last approval timestamp - first approval timestamp
+```
 
-The time it takes to have approves. Once a pull request is opened, how long does it take to have an approve and how long does it take to the last approve?
+### Time from last approval to merge
 
-These metrics are calculated as the time between the pull request being created and the first and last approves being submitted before the merge of the pull request.
+How long does it take to merge a pull request after the last approval:
 
-### Time to first/last review
+```text
+pull request merge timestamp - last approval timestamp
+```
 
-The time it takes to have reviews. Once a pull request is opened, how long does it take to have a review and how long does it take to the last review?
+### Outlier review details
 
-These metrics are calculated as the time between the pull request being created and the first and last reviews being submitted before the merge of the pull request.
+Pulse displays the following details for each pull request in the chart **Review time explorer**:
 
-Reviews include approves, change requests, and inline pull request comments, but don't include pull request conversation comments.
+| Name           | Description                                                               |
+| -------------- | ------------------------------------------------------------------------- |
+| Time to review | Time to merge a pull request                                              |
```

Comment:
Shouldn't we change this? I remember discussing about this, I thought we had changed it. In the product I see time to review, on hover.

Reply:
It's interesting indeed, on the UI copy right next to the chart we're talking about **Time to merge** (I like this because it's the same name we're using on the two charts at the top). But on the tooltip, it's **<time> to review**. :thinking:

@peloycosta If you agree, I'll open a pull request to change the name of the metric on the tooltip to **<time> to merge**.